kubectl-fdb/cmd/restart.go

```diff
@@ -105,10 +106,14 @@ func newRestartCmd(streams genericclioptions.IOStreams) *cobra.Command {
 					processes = append(processes, pod.Name)
 				}
 			} else if len(conditions) > 0 {
-				processes, err = getAllPodsFromClusterWithCondition(kubeClient, clusterName, namespace, conditions)
+				var warn []string
+				processes, warn, err = getAllPodsFromClusterWithCondition(kubeClient, clusterName, namespace, conditions)
 				if err != nil {
 					return err
 				}
+				if warn != nil {
```

Comment:
This will be removed but in general I would prefer a `len(warn) > 0` instead of a nil check.

Reply:
ack